Bump version to v0.6.0 (#954)
open-mmlab/mmengine: README.md

## What's New

> v0.5.0 was released on 2023-02-23.

Reviewer: ```suggestion
v0.6.0 was released on 2023-02-24.
```

Highlights:

> - Add `ApexOptimWrapper`

Reviewer: ```suggestion
- Support `Apex` with `ApexOptimWrapper`
```

- Support analyzing model complexity.
- Add `Lion` optimizer.

> - Support use environment variable in the config file.

Reviewer: ```suggestion
- Support using environment variables in the config file.
```

> - [Support analyzing model complexity](https://mmengine.readthedocs.io/zh_CN/latest/common_usage/model_analysis.html)
> - [Add `Lion` optimizer](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/optim_wrapper.html#id5)

Reviewer: ```suggestion
- [Support analyzing model complexity](https://mmengine.readthedocs.io/en/latest/common_usage/model_analysis.html)
- [Add `Lion` optimizer](https://mmengine.readthedocs.io/en/latest/tutorials/optim_wrapper.html#id5)
```

> Read [Changelog](./docs/en/notes/changelog.md#v060-02232023) for more details.

Reviewer: ```suggestion
Read [Changelog](./docs/en/notes/changelog.md#v060-02232024) for more details.
```

> Read [Changelog](./docs/en/notes/changelog.md#v060-02232024) for more details.

Reviewer: ```suggestion
Read [Changelog](./docs/en/notes/changelog.md#v060-02242023) for more details.
```
